clusterloader2/testing/experiments/ignore_known_gce_container_restarts.yaml

```diff
@@ -10,6 +10,7 @@ RESTART_COUNT_THRESHOLD_OVERRIDES: |
   prometheus-to-sd-exporter: 2
   coredns: 2
   konnectivity-agent: 2
+  node-problem-detector: 2
```

Comment:
Do we know why it started failing recently?

@mborsz @tosi3k - FYI

Reply:
I cannot see the node problem detector log or the kubelet.log and containerd.log on the node. I cannot dig into it.

Reply:
> Do we know why it started failing recently?

I find one OOM in one of the failing case: however, the OOM killed metric-server process which has no memory limit or request. It indicates node memory is full during the running. 

So I prefer to think node problem detector pod restarted for reasons like node resource or node restart.
- We need more logs to know what happened. And the exit code is 255(killed by other processes) in once log which means it is not related to NPD itself.